net/gallery-dl/Portfile

```diff
@@ -0,0 +1,61 @@
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                gallery-dl
+version             1.19.3
+categories          net
+maintainers         nomaintainer
+revision            0
+
+checksums           rmd160  a13818aa36cced652611d66046bed81b65285637 \
+                    sha256  db3973a17f1074e4f4cb99b635b94c1926bdd549263e1df909498f3c6aa93484 \
+                    size    389787
+
+description         command-line program to download image galleries and collections from several image hosting sites
+long_description    ${name} is a {*}${description} It is a cross-platform tool with many configuration options and powerful filenaming capabilities.
+
+homepage            https://github.com/mikf/gallery-dl
+platforms           darwin
+supported_archs     noarch
+license             GPL-2
+
+github.setup        mikf gallery-dl 1.19.3 v
+github.tarball_from releases
+distname            gallery_dl-${github.version}
+depends_lib         port:py${python.version}-requests
+
+variant ffmpeg description {Add ffmpeg dependency to enable Pixiv Ugoira to WebM conversion} { depends_run-append port:ffmpeg }
+
+variant youtubedl conflicts python34 python310 description {Add youtube-dl dependency to enable video downloads} { depends_run-append port:youtube-dl }
+variant ytdlp     conflicts python34 python310 description {Add yt-dlp dependency to enable video downloads}     { depends_run-append port:yt-dlp }
+
+variant python34  conflicts python35 python36 python37 python38 python39 python310 description {Use Python 3.4}  {}
+variant python35  conflicts python34 python36 python37 python38 python39 python310 description {Use Python 3.5}  {}
+variant python36  conflicts python34 python35 python37 python38 python39 python310 description {Use Python 3.6}  {}
+variant python37  conflicts python34 python35 python36 python38 python39 python310 description {Use Python 3.7}  {}
+variant python38  conflicts python34 python35 python36 python37 python39 python310 description {Use Python 3.8}  {}
+variant python39  conflicts python34 python35 python36 python37 python38 python310 description {Use Python 3.9}  {}
+variant python310 conflicts python34 python35 python36 python37 python38 python39  description {Use Python 3.10} {}
+
+if {![variant_isset python34] && ![variant_isset python35] && ![variant_isset python36] && ![variant_isset python37] && ![variant_isset python38] && ![variant_isset python39] && ![variant_isset python310]} {
+    default_variants +python39
+}
+
+if {[variant_isset python34]} {
+    python.default_version 34
+} elseif {[variant_isset python35]} {
+    python.default_version 35
+} elseif {[variant_isset python36]} {
+    python.default_version 36
+} elseif {[variant_isset python37]} {
+    python.default_version 37
+} elseif {[variant_isset python38]} {
+    python.default_version 38
+} elseif {[variant_isset python39]} {
+    python.default_version 39
+} elseif {[variant_isset python39]} {
+    python.default_version 310
+}
```

Comment:
in fact, is there any reason for all these variants - why not just pick one version (i.e., Python 3.9 - the current default in the `python` PG) and just use that. It seems that this is a program that happens to be written in Python, not intended to be imported as a package... or?